### PR TITLE
Embeds an OpenStreetMap via Leaflet

### PR DESCRIPTION
**Pull request by Morpheu5**
Somehow marker images don't seem to work. They appear to be loaded as base64 inlines, and I'm not sure what's wrong with it.

### CODE REVIEW

File: src/app/components/pages/EventDetails.tsx [hunk 1]
```diff
@@ -126,6 +129,14 @@ export const EventDetails = ({match: {params: {eventId}}, location: {pathname}}:
             setBookingFormOpen(true);
         }
 
+        // This is UGLY but there's a weird issue between the leaflet.css file and how webpack loads url()s that makes everything go kaboom.
```
Comment (bh412) [inline src/app/components/pages/EventDetails.tsx:132]: Would you mind adding a link to that suggested fix, I saw it too, so it's documented in future?

File: config/webpack.config.common.js [hunk 2]
```diff
@@ -18,7 +18,7 @@ module.exports = (isProd) => {
...
+                "Content-Security-Policy-Report-Only": "default-src 'self' https://cdn.isaacphysics.org https://cdn.isaaccomputerscience.org localhost:8080 ws://localhost:8080 https://www.google-analytics.com https://maps.googleapis.com; object-src 'none'; frame-src 'self' https://editor.isaaccode.org https://anvil.works https://*.anvil.app https://www.youtube-nocookie.com; img-src 'self' localhost:8080 data: https://cdn.isaacphysics.org https://cdn.isaaccomputerscience.org https://www.google-analytics.com https://i.ytimg.com https://maps.googleapis.com https://maps.gstatic.com https://*.tile.openstreetmap.org; style-src 'self' 'unsafe-inline' https://fonts.googleapis.com; font-src 'self' https://cdn.isaacphysics.org https://cdn.isaaccomputerscience.org https://fonts.gstatic.com;",
```
Comment (bh412) [inline config/webpack.config.common.js:21]: Are we OK to just alter this, could it accidentally have negative implications or is it just loading the map?

**Review by bh412 — COMMENTED** [review]
Just a few requests / clarifications

File: config/webpack.config.common.js (hunk 2 above)
Comment (Morpheu5) [reply config/webpack.config.common.js:21]: Just loading the map. I was hesitant too but I saw we used to have an entry for GMaps too (which this PR removes…)

Comment (bh412) [reply config/webpack.config.common.js:21]: Should we remove the gmaps entry if it's no longer needed?

File: src/scss/phy/isaac.scss [hunk 3]
```diff
@@ -201,3 +202,9 @@ body {
 .footerLogo {
     margin-top: -0.55px;
 }
+
+.leaflet-container {
+    width: 100%;
+    // height: 100vh;
```
[the remark this answers is not in the document]
Comment (Morpheu5) [reply src/scss/phy/isaac.scss]: I moved this to `common/isaac.scss`

File: src/app/components/pages/EventDetails.tsx (hunk 1 above)
Comment (Morpheu5) [reply src/app/components/pages/EventDetails.tsx:132]: Done.

File: config/webpack.config.common.js (hunk 2 above)
Comment (Morpheu5) [reply config/webpack.config.common.js:21]: I did remove it already, didn't I?

Comment (Morpheu5) [reply config/webpack.config.common.js]: I hadn't. Now I have.

**Pull request merged**